services/queue/src/workclaimer.js

```diff
@@ -228,7 +123,7 @@ class WorkClaimer extends events.EventEmitter {
     // put the claim-expiration message into the queue first.  If the
     // subsequent claim_task fails, the claim-expiration message will be
     // ignored when it appears.
-    await this._queueService.putClaimMessage(taskId, runId, takenUntil);
+    await this._queueService.putClaimMessage(taskId, runId, takenUntil, task.taskQueueId, workerGroup, workerId);
```

Comment:
storing extra info in claim table, so we could also answer those questions in future:
- how many tasks are currently running for task queue, worker group, worker